### ISSUE TITLE
The input data

### ISSUE THREAD

**Issue by hwei-hw** [abhi4ssj/relaynet_pytorch#2]
@abhi4ssj Could  you open the 'Data.h5'? Thanks very much!!

**Comment by Atomwh** [comment]
whether the trained PyTorch model can deal the image with different shape with train image ? for example, the train image's shape is 512*64*1 and the test image is 496*64*1. is it right ?

**Comment by zhewang95** [comment]
I think yes, the paper also say that.

**Comment by zhewang95** [comment]
Sure, I'd like to. can't see your email, can you email me your wechat or what you like?

**Comment by Atomwh** [comment]
@zhewang95 Thanks very much ! I will send a email for my WeChat

**Comment by abhi4ssj** [comment]
@Atomwh 
As the network is fully convolutional, it can handle test data of any size. I changed the data size to 512 rows by padding (power of 2), which makes it easy to track the intermediate feature map size. 
Also note that the data slicing code is not yet implemented in this version. It is in the original code in my repo.
